### DISCUSSION TITLE
Help -  WOL + Docker + WebOS

### DISCUSSION THREAD

**Discussion by echopage** [indomus/forum#68] [HUB: Home Assistant]
Salve, spero qualcuno possa aiutarmi, ho la versione di Home Assistant (ho un po di confusione sulle varie versioni penso la Supervised...visto che ho gli add-on) installata in Docker su NUC Debian..... come letto su vari forum ho il problema del WOL che non dialoga esternamente, e quindi non riesco a far accendere da HA la mia TV LG o  a "controllare" i pc di casa, ho trovato alcune guide che fanno riferimento a Docker Compose o a Reverse Proxy.... Ma essendo una capra in questo campo non vorrei fare danni. Qualcuno può darmi delle dritte da seguire? Grazie in anticipo

**Comment by NdR91** [comment]
Il fatto di aver Supervised su Docker non è una discriminante per il WoL (in teoria). Io avevo la tua stessa configurazione, prima dì passare a core, e non avevo problemi.

Il PC lo accendi in WoL da altri dispositivi?

**Reply by snakuzzo** [reply]
vuole accendere la TV LG.

Domanda...hai abilitato il [WOL](https://www.home-assistant.io/integrations/wake_on_lan/#configuration) su HA prima di configurarlo ?

**Reply by NdR91** [reply]
Anche il PC dice, cosa che avvalora ancora di più la tua teoria.

**Reply by Giancky79** [reply]
Mi aggrego, ma è sotto la stessa rete?

**Reply by echopage** [reply]
Sono tutti sotto la stessa rete, WOL abilitato

```
wake_on_lan:

api:

input_boolean:
```
per provare a accendere la tv con webos

ho inserito oltre che il MAC anche il broadcast_address ma niente, una volta che è acceso lo controllo regolarmente, e riesco anche a spegnerlo, ma non riesco ad accenderlo....

```
  turn_on_action:
    service: wake_on_lan.send_magic_packet
    data:
      mac: xx:xx:xx:xx:xx:xx
      broadcast_address: 192.168.1.255
```

**Reply by fizzi** [reply]
Sei sicuro che la tv resta accesa anche da spenta ? ... io il mio samsung F7000 ho la stessa situazione ... riesco a spegnerlo ma non accenderlo ... ma è normale perchè quando si spegne  non lo pingo più.

**Reply by echopage** [reply]
Quando avevo hassio sul raspberry l'accendevo regolarmente... da quando sono passato su Docker non riesco più ad accenderlo.... mmmm ora che mi ci fai pensare contemporaneamente sono passato conla tv da lan a wifi può essere?

**Reply by snakuzzo** [reply]
Che io sappia...le TV WebOS in wifi quando le spegni perdono la connessione

**Reply by echopage** [reply]
Ok provo a fare il collegamento con la lan e vediamo che succede!!!! Grazie per ora

**Reply by echopage** [reply]
Ok ora funziona il problema era il wifi come detto da snakuzzo! Fatto il Ping con wifi e LAN con il primo, con tv spenta, è irraggiungibile, il secondo è raggiungibile, quindi messo sotto LAN funziona!!
